src/NServiceBus.NHibernate.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -65,73 +72,65 @@ public partial class PersistenceTestsConfiguration
-                    x.Dialect<Oracle10gDialect>();
-                    x.Driver<OracleManagedDataClientDriver>();
-                    x.ConnectionString = System.Environment.GetEnvironmentVariable("OracleConnectionString");
-                }));
+                sagaVariants.Add(CreateVariant("Oracle", DatabaseVariant.Oracle));
             }
             SagaVariants = sagaVariants.ToArray();
 
             var outboxVariants = new List<object>();
             if (sqlConnectionString != null)
             {
-                outboxVariants.Add(CreateVariant("SQL Server Optimistic Native default OutboxRecord", x =>
-                    {
-                        x.Dialect<MsSql2012Dialect>();
-                        x.ConnectionString = sqlConnectionString;
-                    }));
-                outboxVariants.Add(CreateVariant("SQL Server Pessimistic Native default OutboxRecord", x =>
-                    {
-                        x.Dialect<MsSql2012Dialect>();
-                        x.ConnectionString = sqlConnectionString;
-                    }, true, false));
-                outboxVariants.Add(CreateVariant("SQL Server Pessimistic TransactionScope default OutboxRecord", x =>
-                    {
-                        x.Dialect<MsSql2012Dialect>();
-                        x.ConnectionString = sqlConnectionString;
-                    }, true, true));
-                outboxVariants.Add(CreateVariant("SQL Server Optimistic TransactionScope default OutboxRecord", x =>
-                    {
-                        x.Dialect<MsSql2012Dialect>();
-                        x.ConnectionString = sqlConnectionString;
-                    }, false, true));
+                outboxVariants.Add(CreateVariant("SQL Server Optimistic Native default OutboxRecord", DatabaseVariant.SqlServer));
+                outboxVariants.Add(CreateVariant("SQL Server Pessimistic Native default OutboxRecord", DatabaseVariant.SqlServer, true, false));
+                outboxVariants.Add(CreateVariant("SQL Server Pessimistic TransactionScope default OutboxRecord", DatabaseVariant.SqlServer, true, true));
+                outboxVariants.Add(CreateVariant("SQL Server Optimistic TransactionScope default OutboxRecord", DatabaseVariant.SqlServer, false, true));
             }
             OutboxVariants = outboxVariants.ToArray();
         }
 
-        static TestFixtureData CreateVariant<T>(string description, Action<DbIntegrationConfigurationProperties> configureDb, bool pessimistic = false, bool transactionScope = false)
+        public static void ConfigureDb(DatabaseVariant databaseVariant, DbIntegrationConfigurationProperties props)
+        {
+            if (databaseVariant == DatabaseVariant.Oracle)
+            {
+                props.Dialect<Oracle10gDialect>();
+                props.Driver<OracleManagedDataClientDriver>();
+                props.ConnectionString = System.Environment.GetEnvironmentVariable("OracleConnectionString");
```

Comment:
Why the namespace using?

Reply:
Because `Environment` refers to the NHibernate's class